pkg/controller/v1alpha1/inferencegraph/openshift_route_reconciler.go

```diff
@@ -0,0 +1,86 @@
+package inferencegraph
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	v1 "github.com/openshift/api/route/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	log2 "sigs.k8s.io/controller-runtime/pkg/log"
+
+	"github.com/kserve/kserve/pkg/apis/serving/v1alpha1"
+)
+
+type OpenShiftRouteReconciler struct {
+	Scheme *runtime.Scheme
+	Client client.Client
+}
+
+func (r *OpenShiftRouteReconciler) Reconcile(ctx context.Context, inferenceGraph *v1alpha1.InferenceGraph) (string, error) {
+	logger := log2.FromContext(ctx, "subreconciler", "OpenShiftRoute")
```

Comment:
log2 seems something weird to me. is there a reason you use log2 instead of just log?

Reply:
There was a conflict with this variable: https://github.com/opendatahub-io/kserve/blob/928f4fe08b89d2fbfb437322d6ac67082eef222d/pkg/controller/v1alpha1/inferencegraph/knative_reconciler.go#L47

But I agree the naming is not good. I renamed to `ctrlLog`.